source/locale/fr/LC_MESSAGES/content/nutzen/api-nutzen.po

```diff
@@ -25,252 +26,491 @@ msgstr ":fa:`home` :doc:`Manuel <../../index>` :fa:`chevron-right`"
+
+#: ../../source/content/nutzen/api-nutzen.rst:311
+msgid "Auflistung der Kategorien"
+msgstr "Liste des catégories"
+
+#: ../../source/content/nutzen/api-nutzen.rst:319
+msgid "Auflistung der zehn häufigsten Schlagwörter"
+msgstr "Liste des dix mots-clés les plus fréquents"
+
+#: ../../source/content/nutzen/api-nutzen.rst:327
+msgid "Geordnete Schlagwortsuche in den Datasets"
+msgstr "Recherche des mots-clés par fréquence croissante"
+
+#: ../../source/content/nutzen/api-nutzen.rst:332
+msgid "Typische Anwendungen für Datenpublizierende"
+msgstr "Applications typiques pour les personnes publiant des données"
+
+#: ../../source/content/nutzen/api-nutzen.rst:338
+msgid "Harvester-ID Ihrer Organisation finden"
+msgstr "Trouver l'identifiant Harvester de son organisation"
+
+#: ../../source/content/nutzen/api-nutzen.rst:346
+msgid "Liste der manuell hinzugefügten Daten Ihrer Organisation"
+msgstr "Liste des données ajoutées manuellement"
+
+#: ../../source/content/nutzen/api-nutzen.rst:354
+msgid "Liste der geharvesteten Daten Ihrer Organisation"
+msgstr "Liste des données harvestées de votre organisation"
+
+#: ../../source/content/nutzen/api-nutzen.rst:361
+msgid "Wenn Sie über die entsprechenden Rechte verfügen, können Sie die Metadaten Ihrer "
+"Datasets auch über die API-Schnittstelle anpassen. Sie finden Ihren API-Key auf "
+"der Nutzerseite auf dem Backend von opendata.swiss. Die genaue Syntax ist u.a. "
+"von Ihrem spezifischen Betriebssystem abhängig (beispielsweise "
+"das richtige Format der Anführungszeichen)"
+msgstr "Si vous disposez des droits nécessaires, vous pouvez également adapter les "
+"métadonnées de vos datasets via l'interface API. Vous trouverez votre clé API sur "
+"la page utilisateur sur le backend d'opendata.swiss. La syntaxe exacte dépend entre "
+"autres de votre système opérateur spécifique (par exemple le format correcte des "
+"guillements)."
+
+#: ../../source/content/nutzen/api-nutzen.rst:364
+msgid "Senden (POST) über die API Schnittstelle: package_patch"
+msgstr "Envoi (POST) via l'interface API: package_patch"
+
+#: ../../source/content/nutzen/api-nutzen.rst:369
+msgid "Datasets können auch über die API neu erstellt werden. Dazu benötigen Sie Ihren "
+"API-Key und müssen die Metadaten als im JSON-Format übermitteln."
+msgstr "Les datasets peuvent également être créés via l'API. Pour cela, vous avez besoin "
+"de votre clé API et vous devez transmettre les métadonnées au format JSON."
+
+#: ../../source/content/nutzen/api-nutzen.rst:372
+msgid "Senden (POST) über die API Schnittstelle: package_create"
+msgstr "Envoi (POST) via l'interface API: package_patch"
+
+#: ../../source/content/nutzen/api-nutzen.rst:450
+msgid "Bitte beachten Sie, dass die Nutzungsbedingungen auf Ebene der Ressource dem "
+"kontrollierten Vokabular entsprechen. Vgl. Sie dazu den "
+"`DCAT AP CH Standard <https://handbook.opendata.swiss/de/content/glossar/bibliothek/dcat-ap-ch.htmldct-rights-dcat>`__."
```

Comment:
Please change to <https://handbook.opendata.swiss/de/content/glossar/bibliothek/dcat-ap-ch.html#dct-rights-dcat>